HttpFs/HttpFs.fs

```diff
@@ -1060,16 +1060,25 @@ module Client =
         body             = response.GetResponseStream()
         luggage          = Some (upcast response) }
 
-  /// Sends the HTTP request and returns the full response as a Response record, asynchronously.
-  let getResponse request = job {
-    let webRequest, exec = toHttpWebRequest HttpFsState.empty request
-    do! exec
-    let! resp = getResponseNoException webRequest
-    let wrapped =
+  let tryGetResponse request =
```

Comment:
Hmm... This makes me think that, although it can be reasoned to be the case, the point about commit points being "at the end" in `Alt.fromBeginEnd`, `Alt.fromAsync` and `Alt.fromTask` should perhaps be made more explicit in the Hopac docs.